.circleci/config.yml

```diff
@@ -131,24 +131,28 @@ jobs:
       - attach_workspace:
           at: ~/plotly.js
       - run:
-          name: Update plot-schema.json
-          command: node tasks/bundle_plot_schema.js
+          name: Build dist/
+          command: npm run build
       - store_artifacts:
-          path: build/plotly.js
-          destination: /plotly.js
-      - store_artifacts:
-          path: dist/plotly.min.js
-          destination: /plotly.min.js
+          path: dist
+          destination: dist
+      - run:
+          name: Pack tarball
+          command: |
+            npm pack
+            version=$(node -e "console.log(require('./package.json').version)")
```

Comment:
Could you please apply something like `| cut -c1-7` to the result of `console.log` to reduce the chance of running errors if any?
<pre>version=$(node -e "console.log(require('./package.json').version)" | cut -c1-7)</pre>